src/models/generated/Source.jl

```diff
@@ -24,19 +27,22 @@ This file is auto-generated. Do not edit.
 
 An infinite bus with a constant voltage output.
 
-Commonly used in dynamics simulations to represent a very large machine on a single bus
+Commonly used in dynamics simulations to represent a very large machine on a single bus or for the representation of import/exports in operational simulations
 
 # Arguments
 - `name::String`: Name of the component. Components of the same type (e.g., `PowerLoad`) must have unique names, but components of different types (e.g., `PowerLoad` and `ACBus`) can have the same name
 - `available::Bool`: Indicator of whether the component is connected and online (`true`) or disconnected, offline, or down (`false`). Unavailable components are excluded during simulations
 - `bus::ACBus`: Bus that this component is connected to
-- `active_power::Float64`: Initial active power set point of the unit in MW. For power flow, this is the steady state operating point of the system. For production cost modeling, this may or may not be used as the initial starting point for the solver, depending on the solver used
-- `reactive_power::Float64`: Initial reactive power set point of the unit (MVAR)
-- `R_th::Float64`: Source Thevenin resistance, validation range: `(0, nothing)`
-- `X_th::Float64`: Source Thevenin reactance, validation range: `(0, nothing)`
+- `active_power::Float64`: (default: `0.0`) Initial active power set point of the unit in MW. For power flow, this is the steady state operating point of the system. For production cost modeling, this may or may not be used as the initial starting point for the solver, depending on the solver used
+- `reactive_power::Float64`: (default: `0.0`) Initial reactive power set point of the unit (MVAR)
+- `active_power_limits::MinMax`: (default: `(min=0.0, max=0.0)`) Minimum and maximum stable active power levels (MW)
+- `reactive_power_limits::Union{Nothing, MinMax}`: (default: `(min=0.0, max=0.0)`) Minimum and maximum reactive power limits. Set to `Nothing` if not applicable
+- `R_th::Float64`: (default: `0.0`) Source Thevenin resistance, validation range: `(0, nothing)`
```

Comment:
documentation nitpick: link to a definition of this, unless everyone but me is going to know what it is